site/category.xml

```diff
@@ -3,49 +3,93 @@
    <description>
       To install these features, point Eclipse at this site.
    </description>
-   <feature id="org.fusesource.ide.core.feature">
-      <category name="AllFeatures"/>
-   </feature>
-   <feature id="org.fusesource.ide.core.feature.source">
-      <category name="AllSources"/>
-   </feature>
-
-   <feature id="org.fusesource.ide.camel.editor.feature">
-      <category name="AllFeatures"/>
-   </feature>
-   <feature id="org.fusesource.ide.camel.editor.feature.source">
-      <category name="AllSources"/>
-   </feature>
-
-   <feature id="org.fusesource.ide.jmx.feature">
-      <category name="AllFeatures"/>
-   </feature>
-   <feature id="org.fusesource.ide.jmx.feature.source">
-      <category name="AllSources"/>
-   </feature>
-
-   <feature id="org.fusesource.ide.server.extensions.feature">
-      <category name="AllFeatures"/>
-   </feature>
-   <feature id="org.fusesource.ide.server.extensions.feature.source">
-      <category name="AllSources"/>
-   </feature>
-
-   <feature id="org.jboss.tools.fuse.transformation.feature">
-      <category name="AllFeatures"/>
-   </feature>
-   <feature id="org.jboss.tools.fuse.transformation.feature.source">
-      <category name="AllSources"/>
-   </feature>
-
-   <category-def name="AllFeatures" label="JBoss Fuse Tooling Nightly Build Update Site">
+
+   <!-- features -->
+
+   <feature id="org.fusesource.ide.camel.editor.feature"><category name="AllFeatures"/></feature>
+   <feature id="org.fusesource.ide.core.feature"><category name="AllFeatures"/></feature>
+   <feature id="org.fusesource.ide.jmx.feature"><category name="AllFeatures"/></feature>
+   <feature id="org.fusesource.ide.server.extensions.feature"><category name="AllFeatures"/></feature>
+   <feature id="org.jboss.tools.fuse.transformation.feature"><category name="AllFeatures"/></feature>
+
+   <!-- tests -->
+
+   <bundle id="org.fusesource.ide.camel.editor.tests"><category name="AllTests"/></bundle>
+   <!-- <bundle id="org.fusesource.ide.camel.editor.tests.integration"><category name="AllTests"/></bundle> -->
```

Comment:
Note: I've commented out the tests that don't pass locally so that this PR can build OK on my machine.